packages/client/src/containers/LandingPage/LandingPage.Container.js

```diff
@@ -1,25 +1,85 @@
-import React, { useEffect, useState } from 'react';
-import { apiURL } from '../../apiURL';
-import './LandingPage.Style.css';
+// import React, { useEffect, useState } from 'react';
+// import { apiURL } from '../../apiURL';
+// import './LandingPage.Style.css';
+
+// export const LandingPage = () => {
+//   const [exampleResources, setExampleResources] = useState([]);
+//   useEffect(() => {
+//     async function fetchExampleResources() {
+//       const response = await fetch(`${apiURL()}/exampleResources`);
+//       const examples = await response.json();
+//       setExampleResources(examples);
+//     }
 
-export const LandingPage = () => {
-  const [exampleResources, setExampleResources] = useState([]);
-  useEffect(() => {
-    async function fetchExampleResources() {
-      const response = await fetch(`${apiURL()}/exampleResources`);
-      const examples = await response.json();
-      setExampleResources(examples);
-    }
+//     fetchExampleResources();
+//   }, []);
 
-    fetchExampleResources();
-  }, []);
+//   return (
+//     <div className="landing-page-container">
+//       <span>Landing Page</span>
+//       {exampleResources.map((example) => (
+//         <div key={example.id}>{example.title}</div>
+//       ))}
+//     </div>
+//   );
+// };
+
+import React from 'react';
+import Testimonials from '../../components/Login/Testimonials';
+import logo from './logo.svg';
+import { Typography, TextField, Button } from '@mui/material';
+import styled from '@emotion/styled';
+import './LandingPage.Style.css';
+// Import necessary components from Material-UI
 
+const CustomButton = styled(Button)({
```

Comment:
It is always good to have styled components, however, not many big companies will be using this library, because it is very difficult to mantain.
As simple as when I am trying to edit the style of an element, I usually go to the css files. If some of them will have styles in the `.jsx` files it will create some confusions.

What I would suggest is to create your own styled button, mixing Material UI and css, if you want to override the style every time:
```
// src/custom-components/Button/Button.jsx

Button.propTypes = {
  onClick: PropTypes.function.isRequired,
  variant: PropTypes.string.isRequired,
  // etc.
};

export const Button = ({ onClick, variant, children }) => {
 
  return (
    <Button onClick={onClick} className={`button-${variant}`}>{children}</Button>
  );
};

// And the css next to it:
// src/custom-components/Button/button.css

.button-primary {
  // custom-css
}

.button-secondary {
  // ...
}
```

If it is a one time style, only applied here you can just simple add the `className="custom-css"` to the material UI component.